ddtrace/utils/http.py

```diff
@@ -7,3 +10,25 @@ def normalize_header_name(header_name):
     :rtype: str
     """
     return header_name.strip().lower() if header_name is not None else None
+
+
+def sanitize_url_for_tag(url):
```

Comment:
I think we should pull this change out into a separate PR. It'd also be great to use it by default in `trace_utils.set_http_meta`.

Reply:
Makes sense. I'll prepare the PR with this change